tests/para-tests-no-ci/test-xcm-para.ts

```diff
@@ -94,11 +120,527 @@ describeParachain(
         ((await relayOne.query.system.account(aliceRelay.address)) as any).data.free.toHuman()
       ).to.eq("8.9999 kUnit");
       // Alith asset balance should have been increased to 1000*e12
-      console.log((await parachainOne.query.assets.account(assetId, ALITH)).toHuman().balance);
       expect(
         (await parachainOne.query.assets.account(assetId, ALITH)).toHuman().balance ===
           "1,000,000,000,000,000"
       ).to.eq(true);
     });
   }
 );
+
+describeParachain("XCM - send_relay_asset_to_relay", { chain: "moonbase-local" }, (context) => {
+  let keyring: Keyring,
+    aliceRelay: KeyringPair,
+    alith: KeyringPair,
+    baltathar: KeyringPair,
+    parachainOne: ApiPromise,
+    relayOne: ApiPromise,
+    assetId: string;
+  before("First send relay chain asset to parachain", async function () {
+    keyring = new Keyring({ type: "sr25519" });
+
+    // Setup Relaychain
+    aliceRelay = keyring.addFromUri("//Alice");
+    relayOne = context._polkadotApiRelaychains[0];
+
+    // Setup Parachain
+    alith = await keyring.addFromUri(ALITH_PRIV_KEY, null, "ethereum");
+    baltathar = await keyring.addFromUri(BALTATHAR_PRIV_KEY, null, "ethereum");
+    parachainOne = context.polkadotApiParaone;
+
+    // Log events
+    logEvents(parachainOne, "PARA");
+
+    logEvents(relayOne, "RELAY");
+
+    await new Promise((res) => setTimeout(res, 10000));
+
+    // PARACHAIN A
+    // registerAsset
+    ({ assetId } = await registerAssetToParachain(parachainOne, alith));
+
+    // RELAYCHAIN
+    // Transfer 1000 units to para a baltathar
+    await createBlockWithExtrinsicParachain(
+      relayOne,
+      aliceRelay,
+      relayOne.tx.xcmPallet.reserveTransferAssets(
+        { X1: { Parachain: new BN(1000) } },
```

Comment:
I just realized this is based on XCM v0, and we might wanna change this to V1, as we are ready from the runtime perspective. What do you think @joelamouche ?

Reply:
I can help you with that

Reply:
Sure give it a go. To run the tests, run `npm run para-test-no-ci`